server.js

```diff
@@ -27,9 +23,43 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+const Avocado = new mongoose.model('Avocado', {
+    id: Number,
+    date: Date,
+    averagePrice: Number,
+    totalVolume: Number,
+    totalBagsSold: Number,
+    smallBagsSold: Number,
+    largeBagsSold: Number,
+    xLargeBagsSold: Number,
+    region: String
+})
+
+if (process.env.RESET_DATABASE) {
+  const seedDatabase = async () => {
+    await Avocado.deleteMany()
+  
+    avocadoSalesData.forEach(item => {
+      const newAvocado = new Avocado(item);
+      newAvocado.save()
+    })
+  }
+  seedDatabase()
+}
+
+app.get('/avocados', async (req, res) => {
+  const allAvocados = await Avocado.find()
+  res.json(allAvocados)
+})
+
+app.get('/', async (req, res) => {
+  res.send("We are advocates of avocados")
+})
+
+app.get('/avocados/id/:id', async (req, res) => {
```

Comment:
`id` is a default key for your document so you can use `/avocados/:id` as an endpoint URL